lib/mongo/monitoring/event/command_failed.rb

```diff
@@ -53,6 +53,9 @@ class CommandFailed < Mongo::Event::Base
         # @return [ Integer ] request_id The request id.
         attr_reader :request_id
 
+        # @api private
```

Comment:
This actually needs to be public because this is a cross-driver feature delivered for users.

Reply:
done